packages/graph/README.md

```diff
@@ -1,7 +1,28 @@
-# DEV
+# Search indexing for CapTableRegistry BRØK
 
-Make sure the correct CapTableRegistry is sert in config
+The Graph is an indexing protocol for organizing blockchain data and making it easily accessible with GraphQL.
 
-Create a subgraph at https://thegraph.com/legacy-explorer/
+The Graph makes it possible to query data that is difficult to query directly.
 
-Change the NPM command to reflect new subgraph, then run `yarn run brokDev:deploy`
+The Graph learns what and how to index Ethereum data based on subgraph descriptions, known as the subgraph manifest. The subgraph description defines the smart contracts of interest for a subgraph, the events in those contracts to pay attention to, and how to map event data to data that The Graph will store in its database.
+
+Once you have written a `subgraph manifest`, you use the Graph CLI to store the definition in IPFS and tell the indexer to start indexing data for that subgraph.
+
+This diagram gives more detail about the flow of data once a subgraph manifest has been deployed, dealing with Ethereum transactions:
+
+
+## Project overview
+
+| **Folder** | **Description** |
+|--|--|
+| `abis/` | ABIS files for CapTable and CapTableRegistry, ***NB: These files are generated and copy manually*** |
+| `build/` | Generated files from `graph codegen && graph build`, based on the content of `subgraph.yaml` |
+| `docker-compose/` | Specs for running TheGraph locally with Docker compose |
+| `generated/` | Some manually generated files @jon ? |
```

Comment:
The "build" folder contains compiled and ready-to-deploy files such as ABI files for smart contracts, necessary for deploying the subgraph.

The "generated" folder includes TypeScript types and auxiliary files generated from the smart contracts' ABI, mainly used to ensure type safety and facilitate a smoother development experience when working with GraphQL queries or smart contract interactions.